src/samples/DX11/DX11.cpp

```diff
@@ -285,6 +324,7 @@ int main()
     };
     std::unordered_map<StreamHandle, RenderTarget> renderTargets;
     FrameData frameData;
+    int tick = 0;
```

Comment:
No instance-local variables. All logic must be deterministic and based off the frame request data